Extensions/timestamps.css

```
@@ -30,17 +30,26 @@
 
 .xtimestamp {
 	position: relative;
+	top: -9px;
 	color: rgb(168,177,184);
 	text-align: left;
 	font-weight: normal !important;
+	font-size: 11px;
+	margin-bottom: 10px;
+	margin-left: 20px;
+}
+
+.xkit--react .xtimestamp {
+	top: 0;
```

probably better to just make the `top` rule not do anything; also, `color` needs to be overridden

```suggestion
	position: static;
	color: var(--black);
```